app/components/work_packages/activities_tab/journals/item_component/add_reactions.sass

```diff
@@ -2,3 +2,7 @@
   row-gap: var(--base-size-4, 4px)
   @media screen and (max-width: $breakpoint-sm)
     max-width: 200px
+
+.op-add-reactions-button
+  width: var(--control-medium-size)
+  padding: unset
```

Comment:
`var(--control-medium-size)` is intended to define the height of a control. The width is usally defined by the content itself. Actually, you don't need all of that. Just set `px: 1` on the Button directly and it should get squared.

Reply:
Got it, thanks 👍